### DISCUSSION TITLE
Numero progressivo fattura

### DISCUSSION THREAD

**Discussion by qeimada** [fattureincloud/api#133] [Q&A] [answered]
Nelle vecchia API, il numero di fattura era automatico e progressivo. Come funziona con la versione 2? Come faccio a dare un numero automatico?

**Comment by qeimada** [comment, marked as answer]
Ok ho scoperto. Bastava non mettere l'id

**Reply by valmoz** [reply]
Ottimo @qeimada 
chiedo scusa, mi ero perso la richiesta. Sono felice che sia comunque riuscito a risolvere.

Mauro
